File: Backend/Backend/DataTransferObject/College/CollegeGetResponse.cs
namespace Backend.DataTransferObject;

public class CollegeGetResponse
{
    public int Id { get; set; }
    public string Name { get; set; }
    public DateTime FoundationDate { get; set; }
    public string EmailExtension { get; set; }
    public CityDto City { get; set; }
    public ICollection<StudentDto> Students { get; set; }
}

File: client/src/app/interfaces/college/CollegeGetResponse.ts

// GET /College
import {City} from "../city/City";
import {StudentResponse} from "../student/StudentResponse";

export interface CollegeGetResponse {
  id: number;
  name: string;
  foundationDate: Date;
  emailExtension: string;
  city: City;
  student: StudentResponse[];
}
